build/Nuget/Microsoft.NET.Nuget.proj

```diff
@@ -3,17 +3,13 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'Common.props'))\Common.props" />
 
   <PropertyGroup>
-    <NuGetOutDir>$(OutDir)Packages\</NuGetOutDir>
-    <OutDir>$(OutDir)Raw\</OutDir>
+    <InitialOutDir>$(OutDir)</InitialOutDir>
+    <NuGetOutDir>$(OutDir)Packages/</NuGetOutDir>
+    <OutDir>$(OutDir)Raw/</OutDir>
+    <NuPkgOutput>$(NuGetOutDir)%(FileName).$(Version).nupkg</NuPkgOutput>
```

Comment:
FYI. I think the Clean target in this file now needs to work because it worked based on the itemgroup that no longer exists.